packages/transactions/src/messages/web3Extension.spec.ts

```diff
@@ -19,7 +19,7 @@ describe('web3Extension tests', () => {
 
     const res = signatureToWeb3Extension(chain, sender, signature)
     expect(res.path).toBe('ethermint.types.v1.ExtensionOptionsWeb3Tx')
-    expect(Buffer.from(res.message.serializeBinary()).toString('base64')).toBe(
+    expect(Buffer.from(res.message.toBinary()).toString('base64')).toBe(
```

Comment:
why the change from `serializeBinary()`, to `toBinary()`?

Reply:
We switched Protobuf Typescript packages, the new one has renamed methods and various changes, one of which is this method